R/beta_binomial_ab_test.R

```diff
@@ -4,9 +4,7 @@ beta_binomial_ab_test <- function(y, n,
                                   nsim = 1e5,
                                   conf.level = 0.1,
                                   expected_conversion_rate = NULL,
-                                  groups = 1:length(y),
-                                  plot.density = TRUE,
```

Comment:
`plot.density` isn't needed any longer as plotting has been moved to the `plot()` generic.